client-management-service-impl/src/test/resources/schema.sql

```diff
@@ -10,6 +10,7 @@ CREATE TABLE IF NOT EXISTS client_detail(
     	grant_types character varying NOT NULL,
     	auth_methods character varying NOT NULL,
     	status character varying(20) NOT NULL,
+    	additional_config character varying,
```

Comment:
why is the datatype not set to jsonb here? is it not supported in the H2 database?

Reply:
H2 db does not support "jsonb" type but does support "json" but using it is creating some complications so for now I just used text type.